samples/stock/plotoptions/series-datagrouping-simple-approximation/demo.js

```diff
@@ -0,0 +1,43 @@
+Highcharts.stockChart('container', {
+    rangeSelector: {
+        selected: 4
+    },
+    plotOptions: {
+        series: {
+            tooltip: {
+                valueDecimals: 2
+            },
+            dataGrouping: {
+                forced: true,
+                units: [
+                    ['week', [1]]
+                ]
+            }
+        }
+    },
+    series: [{
+        name: 'ADBE default (average)',
+        data: ADBE
+    }, {
+        name: 'ADBE start',
+        data: ADBE,
+        dataGrouping: {
+            approximation: function () {
+                var groupStart = this.dataGroupInfo.start;
+
+                return this.options.data[groupStart][1];
+            }
```

Comment:
Can be simpler :) `approximation` takes an argument (or many if data is e.g. OHLC), which is an array of values:
```js
            approximation: function (data) {
                return data[0];
            },
```